apps/dapp-console/app/components/Header/HeaderTabs.tsx

```diff
@@ -45,20 +48,26 @@ const hoverClasses = 'hover:border-muted-foreground hover:text-foreground'
 const activeClasses = 'border-foreground text-foreground'
 
 type HeaderTabItemProps = {
+  id: string,
   href?: string
   isActive?: boolean
   children: React.ReactNode
 }
 
 const HeaderTabItem = ({
+  id,
   href = '',
   isActive,
   children,
 }: HeaderTabItemProps) => {
+  const handleClick = (id: string) => {
+    trackTopBarClick(id)
+  }
   return (
     <Link
       href={href}
       className={cn(baseClasses, isActive ? activeClasses : hoverClasses)}
+      onClick={() => handleClick(id)}
```

Comment:
instead of the `handleClick` you can probably just inline the track function `() => trackTopBarClick(id)`